src/js/util/ui.js

```diff
@@ -106,13 +105,10 @@ define(function (require, exports) {
      *
      * @param {string} url
      * @param {SyntheticEvent?} event
-     * @param {string} name
      */
-    var openURL = function (url, event, name) {
+    var openURL = function (url, event) {
```

Comment:
Hmm... it's weird that we have `openURL` in `js/util/ui` and also `js/actions/menu`...